packages/replicache/src/replicache-impl.ts

```diff
@@ -1450,8 +1455,27 @@ export class ReplicacheImpl<MD extends MutatorDefs = {}> {
       args: JSONValue | undefined,
     ) => Promise<void | JSONValue>;
 
-    return (args?: Args): Promise<Return> =>
-      this.#mutate(name, mutatorImpl, args, performance.now());
+    return (args?: Args): Promise<Return> => {
+      // DO NOT track CRUD mutations as they do not receive responses from
+      // the server.
+      const trackingData =
+        name === '_zero_crud' ? undefined : this.#zero?.trackMutation();
+
+      const result = this.#mutate(
+        trackingData,
+        name,
+        mutatorImpl,
+        args,
+        performance.now(),
+      );
+
+      if (trackingData) {
```

Comment:
> It is not clear to me how this refactoring prevents the wrapping of the result? We are still returning result from the async function.

@arv - we add the `server` field here where we're no longer in an async funcion.

Reply:
OK. I was referring the the changes to `#mutate`.